ntoskrnl/io/pnpmgr/pnpinit.c

```diff
@@ -38,10 +50,46 @@ NTSTATUS
 NTAPI
 IopInitializeArbiters(VOID)
 {
-     /* FIXME: TODO */
-    return STATUS_SUCCESS;
+    NTSTATUS Status;
+
+    Status = IopPortInitialize();
+    if (!NT_SUCCESS(Status))
+    {
+        DPRINT1("IopInitializeArbiters: Status %X\n", Status);
```

Comment:
Nit: I'd prefer if we printed the name of the failing function instead.

Reply:
Ok